src/controllers/expenseController.js

```diff
@@ -0,0 +1,102 @@
+const {
+  filterExpenses,
+  addExpense,
+  getExpense,
+  deleteExpense,
+  updateExpense,
+  getExpenses,
+} = require('../services/expenses.service');
+const { getUser } = require('../services/users.service');
+
+const getAll = (req, res) => {
+  const { userId, categories, from, to } = req.query;
+
+  if (userId || categories || from || to) {
+    res.statusCode = 200;
+    res.send(filterExpenses(req.query));
+
+    return;
+  }
+
+  const expenses = getExpenses();
+
+  res.statusCode = 200;
+  res.send(expenses);
+};
+
+const add = (req, res) => {
+  const { userId, category, spentAt, title, amount, note } = req.body;
+  const user = getUser(userId);
```

Comment:
The `getUser` function is used to check if a user exists. Ensure that this function handles errors properly and returns `null` or `undefined` when the user is not found, as expected by the subsequent logic.